.registryrc

```diff
@@ -1 +1 @@
-3170b725e1e46fdf0c65844fa0454cea1243c8c7
+d54caf30e8204019a93cc92d3c26bef31248f2ba
```

Comment:
update after https://github.com/hyperlane-xyz/hyperlane-registry/pull/545 and https://github.com/hyperlane-xyz/hyperlane-registry/pull/553 are merged

when this gets updated, it'll resolve the agent-configs test